### ISSUE TITLE
Remove "Draw line" and "Draw a rectangle" from drawing tools when adding a location 

### ISSUE THREAD

**Issue by dpalomino** [Cadasta/cadasta-platform#1608]
### Problem Statement

In the vast majority of cases, only markers and polygons are used in the web UI to add/edit geometries. Partners are sometimes confused by the draw line and draw a rectangle option, and in all cases seen those geometries were just created by mistake. 

It'd be useful for partners to simplify the UI and just offer the polygon and marker options. 

### Context/Use case

See above. 

### User Story 

As a data collector, I want only to use marker and polygon drawing tools when adding or editing locations. 

### Description 

We only need to remove the polyline and rectangle options in the drawing tools when adding a new location or editing a geometry. However, we'll maintain the rectangle option when defining the project boundary.

**Comment by wonderchook** [comment]
@dpalomino I'm hesitant to remove the line drawing tool. There are scenarios where people would want to draw lines that I think we are going to run into. 

The OSM Editor iD does a good job of making this more clear. Maybe something like that would help

![image](https://user-images.githubusercontent.com/67992/27592527-8a566a44-5b09-11e7-9281-9927d54a13c8.png)
****

They then have nice mouse overs that explain what each is fo:

![image](https://user-images.githubusercontent.com/67992/27592568-a8466630-5b09-11e7-9990-eb58a7dedfd2.png)

**Comment by dpalomino** [comment]
Yes, definitely that kind of UI would help to clarify. @clash99, what do you think?

**Comment by clash99** [comment]
Love it - will take a little time to implement so maybe for next sprint?

**Comment by dpalomino** [comment]
It'd be really great :-)

**Comment by clash99** [comment]
Related to #1005

**Comment by dpalomino** [comment]
Updating description and title with the new approach.

**Comment by clash99** [comment]
Related to #1608

**Comment by clash99** [comment]
## Wireframes

- [Link to pdf of all wireframes](https://drive.google.com/file/d/0BzpiEtMtHC3rak12ekFrTnEyeEk/view?usp=sharing)

### Default toolbar changes

![recommendations for default](https://user-images.githubusercontent.com/1119621/29037962-67730a32-7b6b-11e7-9b71-6bbf1353b21e.png)



### Draw toolbar changes

![recommendations for draw](https://user-images.githubusercontent.com/1119621/29037960-67710278-7b6b-11e7-8dcc-6dd11832ac15.png)

### User flows for adding locations through draw toolbar

![add location flow - 1_2](https://user-images.githubusercontent.com/1119621/29037963-6773a6a4-7b6b-11e7-8a5d-fc8000367f11.png)
![add location flow - 2_2](https://user-images.githubusercontent.com/1119621/29037961-6771e6c0-7b6b-11e7-8744-cf9df670c7a1.png)

**Comment by dpalomino** [comment]
@clash99, I think the wireframes are really great. Thanks!

Re the bulk import of locations, I think even if we don't support bulk imports, we need to see how the user will be selecting the location to be imported if the file contains more than one location.
